pokemon-checker/src/components/PokemonDisplay/DisplayFunctionalComponents/TypeDisplay.tsx

```diff
@@ -1,16 +1,95 @@
-type pokemonTypes = {
-  type1: string,
-  type2: string | null
+import { ElementType } from "../../../constants/ElementTypes";
+
+type TypeDisplayProps = {
+  type: ElementType;
+};
+
+const spaceStyles = {
```

Comment:
Structural rules like `margin`, `padding`, and such will remain consistent between all types. This object gets merged together with the resulting colour and then added to the rendered component.